Tests/BoardTest.cs

```diff
@@ -0,0 +1,67 @@
+﻿using GameEngine;
+using GameEngine.Entities;
+using GameEngine.Interfaces;
+using Tests.Animals;
+using Xunit;
+
+namespace Tests
+{
+    public class BoardTest
+    {
+        private Board _board;
+        private List<IItem> _items;
+
+        public BoardTest()
+        {
+            _board = new Board();
+            _items = new List<IItem>
+            {
+                new TestCarnivore
+                {
+                    CoordinateX = 0,
+                    CoordinateY = 0,
+                    Id = 0,
+                    Sex = AnimalSex.Female,
+                    IsPaired = false,
+                    IsVisible = true,
+                    Vision = 2,
+                    Health = 100,
+                    Specie = "Test",
+                    Letter = ConsoleKey.T,
+                    Color = ConsoleColor.White,
+                    IsActive = true
+                }
+            };
+        }
+
+        [Fact]
+        public void CreateArray_CheckArrayLength_ReturnsHundredFifty()
```

Comment:
Comment for all tests, missing AAA, for example
![image](https://user-images.githubusercontent.com/103017380/180126691-24fce739-a104-4dbe-8b73-d45f5624cbb6.png)

Reply:
Done.